### PR TITLE
Add under-utilized container alerts

### PR DESCRIPTION
**Pull request by michaelact**

### CODE REVIEW

**Comment by awcodify** [comment]
I love the idea 👍

File: _data/rules.yml [hunk 1]
```diff
@@ -329,7 +329,7 @@ groups:
                 for: 5m
                 comments: |
                   This rule can be very noisy in dynamic infra with legitimate container start/stop/deployment.
-              - name: Container CPU usage
+              - name: Container CPU utilization
```
Comment (samber) [inline _data/rules.yml:332]: Container High CPU utilization

File: _data/rules.yml [hunk 2]
```diff
@@ -329,7 +329,7 @@ groups:
...
                 description: Container CPU usage is above 80%
                 query: '(sum(rate(container_cpu_usage_seconds_total{name!=""}[3m])) BY (instance, name) * 100) > 80'
                 severity: warning
```
Comment (samber) [inline _data/rules.yml:335]: Container High Memory usage

File: _data/rules.yml [hunk 3]
```diff
@@ -350,6 +350,21 @@ groups:
                 query: 'rate(container_cpu_cfs_throttled_seconds_total[3m]) > 1'
                 severity: warning
                 for: 2m
+          - name: google/cAdvisor
+            slug: google-cadvisor-under-utilized
+            doc_url: https://github.com/google/cadvisor
+            rules:
+              - name: Container CPU utilization
```
Comment (samber) [inline _data/rules.yml:357]: Container Low CPU utilization

File: _data/rules.yml [hunk 4]
```diff
@@ -350,6 +350,21 @@ groups:
...
+                description: Container CPU usage is under 20% for 1 week. Consider reducing memory space.
+                query: '(sum(container_memory_working_set_bytes{name!=""}) BY (instance, name) / sum(container_spec_memory_limit_bytes > 0) BY (instance, name) * 100) < 20'
+                severity: info
+                for: 7d
+              - name: Container Memory usage
```
Comment (samber) [inline _data/rules.yml:362]: Container Low Memory usage

File: _data/rules.yml [hunk 5]
```diff
@@ -350,6 +350,21 @@ groups:
                 query: 'rate(container_cpu_cfs_throttled_seconds_total[3m]) > 1'
                 severity: warning
                 for: 2m
+          - name: google/cAdvisor
+            slug: google-cadvisor-under-utilized
+            doc_url: https://github.com/google/cadvisor
+            rules:
+              - name: Container CPU utilization
+                description: Container CPU usage is under 20% for 1 week. Consider reducing memory space.
+                query: '(sum(container_memory_working_set_bytes{name!=""}) BY (instance, name) / sum(container_spec_memory_limit_bytes > 0) BY (instance, name) * 100) < 20'
```
Comment (samber) [inline _data/rules.yml:359]: this query is identical to the next one

i think you made a small mistake ;)

File: _data/rules.yml [hunk 6]
```diff
@@ -350,6 +350,21 @@ groups:
                 query: 'rate(container_cpu_cfs_throttled_seconds_total[3m]) > 1'
                 severity: warning
                 for: 2m
+          - name: google/cAdvisor
```
Comment (samber) [inline _data/rules.yml:353]: you can skip this header, since this is the same exporter than previous rules

**Review by samber — CHANGES_REQUESTED** [review]

**Pull request merged**